services/blockchain-connector/shared/sdk/events.js

```diff
@@ -53,36 +53,15 @@ const subscribeToAllRegisteredEvents = async () => {
 	});
 };
 
-const getEventsByHeight = async (height) => {
-	try {
-		const chainEvents = await invokeEndpoint('chain_getEvents', { height });
-		const eventsResponse = await BluebirdPromise.map(
-			chainEvents,
-			async (event) => {
-				const schema = await getEventSchemaByName(event.name);
-				const decodedEventData = schema && event.data !== ''
-					? await decodeEvent(event.data, schema) : {};
-				const eventID = await calculateEventID(event);
-
-				return {
-					...decodedEventData,
-					id: eventID,
-				};
-			},
-			{ concurrency: chainEvents.length },
-		);
-
-		return eventsResponse;
-	} catch (err) {
-		if (err.message.includes(timeoutMessage)) {
-			throw new TimeoutException('Request timed out when calling \'chain_getEvents\'');
-		}
-		throw err;
-	}
+const getEventsByHeightDecoded = async (height) => {
```

Comment:
An alternative name could be `getDecodedEventsByHeight`

Reply:
I'd prefer to keep the name as is so that when we search for implementations by the original method's name these methods also show up in the result.

Reply:
This one is already mapped to different name [here](https://github.com/LiskHQ/lisk-service/blob/efa7795cc5039045fc6f4a29ea85f64884ef2035/services/blockchain-connector/shared/sdk/events.js#L66).